src/components/Send/ShowUtxos.tsx

```diff
@@ -0,0 +1,393 @@
+
+type UtxoList = Array<Utxo>
+
+interface ShowUtxosProps {
+  wallet: CurrentWallet
+  show: boolean
+  onHide: () => void
+  index: String
+}
+
+interface UtxoRowProps {
+  utxo: Utxo
+  utxoIndex: number
+  onToggle?: (index: number, isFrozen: boolean) => void
+  isFrozen: boolean
+  settings: Settings
+  showRadioAndBg: boolean
+  walletInfo: WalletInfo
+  t: TFunction
+}
+
+interface UtxoListDisplayProps {
+  utxos: Array<Utxo>
+  onToggle?: (index: number, isFrozen: boolean) => void
+  settings: Settings
+  showRadioAndBg?: boolean
+}
+
+interface DividerProps {
+  isState: boolean
+  setIsState: (arg: boolean) => void
+  className?: string
+}
+
+// Utility function to format Bitcoin address
+const formatAddress = (address: string) => `${address.slice(0, 10)}...${address.slice(-8)}`
+
+// Utility function to format the confirmations
+const formatConfirmations = (conf: number) => {
+  if (conf === 0) return { symbol: 'confs-0', confirmations: conf }
+  if (conf === 1) return { symbol: 'confs-1', confirmations: conf }
+  if (conf === 2) return { symbol: 'confs-2', confirmations: conf }
+  if (conf === 3) return { symbol: 'confs-3', confirmations: conf }
+  if (conf === 4) return { symbol: 'confs-4', confirmations: conf }
+  if (conf === 5) return { symbol: 'confs-5', confirmations: conf }
+  if (conf >= 9999) return { symbol: 'confs-full', confirmations: '9999+' }
+  return { symbol: 'confs-full', confirmations: conf }
+}
+
+// Utility function to convert Satoshi to Bitcoin
+const satsToBtc = (sats: number) => (sats / 100000000).toFixed(8)
+
+// Utility function to Identifies Icons
+const utxoIcon = (tag: string, isFrozen: boolean) => {
+  if (isFrozen && tag === 'bond') return 'timelock'
+  if (isFrozen) return 'snowflake'
+  if (tag === 'deposit' || tag === 'non-cj-change' || tag === 'reused') return 'Unmixed'
+  if (tag === 'bond') return 'timelock'
+  return 'mixed'
+}
+
+// Utility function to allot classes
+const allotClasses = (tag: string, isFrozen: boolean) => {
+  if (isFrozen) return { row: styles.frozenUtxo, tag: styles.utxoTagFreeze }
+  if (tag === 'deposit') return { row: styles.depositUtxo, tag: styles.utxoTagDeposit }
+  if (tag === 'joined' || tag === 'cj-out') return { row: styles.joinedUtxoAndCjout, tag: styles.utxoTagJoinedAndCjout }
+  if (tag === 'non-cj-change' || tag === 'reused')
+    return { row: styles.changeAndReuseUtxo, tag: styles.utxoTagChangeAndReuse }
+  return { row: styles.depositUtxo, tag: styles.utxoTagDeposit }
+}
+
+// Utxos row component
+const UtxoRow = memo(
+  ({ utxo, utxoIndex, onToggle, isFrozen, showRadioAndBg, settings, walletInfo, t }: UtxoRowProps) => {
+    const address = formatAddress(utxo.address)
+    const conf = formatConfirmations(utxo.confirmations)
+    const value = satsToBtc(utxo.value)
+    const tag = utxoTags(utxo, walletInfo, t)
+    const icon = utxoIcon(tag[0].tag, isFrozen)
+    const rowAndTagClass = allotClasses(tag[0].tag, isFrozen)
```

Comment:
Instead of line 103 and 104, add this:
```
let icon, rowAndTagClass
    if (tag.length === 0) {
      icon = 'empty'
      rowAndTagClass = { row: styles.depositUtxo, tag: styles.utxoTagDeposit }
    } else {
      icon = utxoIcon(tag[0].tag, isFrozen)
      rowAndTagClass = allotClasses(tag[0].tag, isFrozen)
    }
```